src/Stepper/Step.less

```diff
@@ -0,0 +1,174 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+@stepIconWidth: 1.75rem; // 28px
+@iconBorderWidth: @size_4xs; // 1px
+@iconFocusShadowThickness: @size_2xs; // 4px
+@titleFontSize: 1.25rem; // 20px
+@alertSize: 0.375rem; // 6px
+
+.Step {
+  .flexbox();
+  .alignItems(flex-start);
+  width: 100%;
+  .padding--y--2xs;
+}
+
+.Step--button {
+  .button--reset();
+  .padding--none;
+  text-align: left;
+  .text--line-height-3;
+}
+
+.Step--title {
+  overflow: hidden;
+  color: @neutral_gray;
+  font-size: @titleFontSize;
+  .text--semi-bold;
+  .margin--top--2xs;
+  -ms-flex: 0 1 auto;
+  transition: color @timingQuickly ease-out;
+}
+
+.Step--description {
+  color: @neutral_gray;
+  .text--medium;
+  .text--regular;
+  transition: color @timingQuickly ease-out;
+}
+
+.Step--message--optional {
+  overflow: hidden;
+  color: @neutral_gray;
+  .text--medium;
+  .text--italic;
+  .margin--bottom--xs;
+  transition: color @timingQuickly ease-out;
+}
+
+.Step--message--warning {
+  overflow: hidden;
+  color: @alert_red;
+  .text--medium;
+  .text--semi-bold;
+  .margin--bottom--xs;
+}
+
+.Step--icon {
+  display: inline-block;
+  .flexShrink(0);
+  .flexbox();
+  .alignItems(center);
+  .justifyContent(center);
+  .circle(@stepIconWidth);
+  background-color: @neutral_white;
+  border: @iconBorderWidth solid @neutral_gray;
+  .text--medium;
+  .text--regular;
+  .margin--right--s;
+  color: @neutral_gray;
+  box-shadow: 0 0 0 @iconFocusShadowThickness transparent;
+  transition: background-color @timingQuickly ease-out, border-color @timingQuickly ease-out;
```

Comment:
lot simpler and more readable to go with `all @timingQuickly ease-out` to avoid listing properties out.

that's usually the go-to unless we explicitly need to animate only specific properties. otherwise, someone could add a property change that doesn't get animated and looks weird in relation to the one that's animating.

https://github.com/Clever/components/search?q=transition%3A+all&unscoped_q=transition%3A+all